socialhome/streams/templatetags/slurp_json_context.py

```diff
@@ -0,0 +1,36 @@
+from django.utils.safestring import mark_safe
+from django.template import Library
+import json
+
+register = Library()
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context(context):
+    """
+    Serializes ``context["json_context"]`` into JSON and generates a JS
+    script to attach it to ``window.context``
+    Args:
+        context: Current view context
+
+    Returns: "<script>window.context = {<context["json_context"]>}</script>"
+
+    """
+    return _slurp_json_context(context, lambda obj: str(obj))
+
+
+@register.simple_tag(takes_context=True)
+def slurp_json_context_unsafe(context):
+    """
+    Has the same behaviour than ``slurp_json_context`` at the exception
+    of throwing an error when a variable can't be serialized.
+    """
+    return _slurp_json_context(context, None)
+
+
+def _slurp_json_context(context, json_default):
+    if context.get("json_context", None) is None:
```

Comment:
`.get()` default is `None` or check against `None` since empty is falsey, so you don't need to specify it, just do `if context.get("json_context"):`.